src/parser/shared/modules/items/bfa/raids/bod/IncandescentSliver.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { calculateSecondaryStatDefault } from 'common/stats';
+import { formatNumber } from 'common/format';
+
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS/index';
+
+import Analyzer from 'parser/core/Analyzer';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import Combatants from 'parser/shared/modules/Combatants';
+import StatisticBox from 'interface/others/StatisticBox';
+import ItemIcon from 'common/ItemIcon';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+
+
+class IncandescentSliver extends Analyzer {
+  critRating = 0;
+  masteryRating = 0;
+
+  static dependencies = {
+    statTracker: StatTracker,
+    combatants: Combatants,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrinket(ITEMS.INCANDESCENT_SLIVER.id);
+    if(this.active) {
+      const item = this.selectedCombatant.getItem(ITEMS.INCANDESCENT_SLIVER.id);
+      this.critRating = calculateSecondaryStatDefault(385, 20, item.itemLevel);
+      this.masteryRating = calculateSecondaryStatDefault(385, 66, item.itemLevel);
+      this.statTracker.add(SPELLS.INCANDESCENT_LUSTER.id, {
+        crit: this.critRating,
+      });
+      this.statTracker.add(SPELLS.INCANDESCENT_BRILLIANCE.id, {
+        mastery: this.masteryRating,
+      });
+    }
+  }
+
+  averageCritRating() {
```

Comment:
This could be a getter too, which would probably be cleaner.

Reply:
Fixed.